bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -1746,20 +1781,35 @@ int inner_map_mng_persist()
     return 0;
 }
 
-void inner_map_mng_restore_by_persist_stat(struct persist_info *p, struct inner_map_stat *stat)
+int inner_map_mng_restore_by_persist_stat(struct persist_info *p, struct inner_map_stat *stat)
 {
     memcpy(g_inner_map_mng.inner_maps, stat, sizeof(struct inner_map_stat) * (p->max_allocated_idx + 1));
 
+    // What is recorded in g_inner_map_mng.inner_maps[i].map_fd is map_id.
+    // Use map_id to get the fd of the inner_map in this process and refresh
+    // it to inner_maps for use.
+    for (int i = 0; i < p->max_allocated_idx; i++) {
+        if (g_inner_map_mng.inner_maps[i].allocated) {
+            int map_fd = bpf_map_get_fd_by_id(g_inner_map_mng.inner_maps[i].map_fd);
+            if (map_fd < 0) {
+                LOG_ERR("restore_by_persist_stat bpf_map_get_fd_by_id failed, i:[%d] map_fd:[%d]\n", i, map_fd);
```

Comment:
what is `i` means

Reply:
i represents the order in inner_map. Their order in inner_map will affect the judgment of expansion and contraction.
Since inner_maps are all nameless maps, an array is used to record their map_id in g_inner_map_mng.

Reply:
get